src/layouts/SideNavigationButton.vue

```diff
@@ -0,0 +1,24 @@
+<script lang="ts" setup>
+interface Props {
+  isHere: boolean
+  path: string
+  text: string
+}
+
+const props = defineProps<Props>()
```

Comment:
Vue 3.5 以降だと、こんな感じで分割代入をするとかなり簡単に props の各フィールドにアクセスできるようになるので使ってみると良いと思います
参考: https://ja.vuejs.org/guide/components/props#reactive-props-destructure
```suggestion
const { isHere, path, text } = defineProps<Props>()
```